lite2/client.go

```diff
@@ -88,6 +90,15 @@ func AlternativeSources(providers []provider.Provider) Option {
 	}
 }
 
+// RemoveNoLongerTrustedHeadersPeriod option can be used to define how often
```

Comment:
> I'm a little confused - do we expect the light client to be running as a daemon?

Yeah. Constantly running on people's servers or mobile phones or smth else

> Can this just be a flag or command to gaiacli [xyz]?

You mean making RemoveNoLongerTrustedHeadersPeriod optional. Totally possible, but most would probably enroll since it will be maintaining constant disk usage (opposite to storing all headers).

Reply:
Ah, more I mean "running every time gaiacli [xyz] is called" (if there are headers which are old).

Reply:
Got it. Will make this optional